src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -281,7 +306,7 @@
    :component-did-mount
    (fn [{:keys [state props refs this]}]
      (endpoints/call-ajax-orch
-       {:endpoint (endpoints/get-workspace-method-config (:workspace-id props) (:config props))
+       {:endpoint (endpoints/get-validated-workspace-method-config (:workspace-id props) (:config props))
```

Comment:
If we go that route, then we can only validate syntax once a config is updated. So we either display syntax validation at all times or we could have a button that explicitly validates the syntax on demand. The former seemed like the better approach to me, because it made it apparent that there were problems with the config by never hiding that information, but I'm fine implementing it either way.